### PR TITLE
PXB-2366 Document issue found for --databases parameter

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/xtrabackup_bin/partial_backups.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst [hunk 1]
```diff
@@ -74,15 +74,21 @@ this list make sure to specify the ``mysql``, ``sys``, and
 ``performance_schema`` databases. These databases are required when restoring
 the databases using :option:`xtrabackup --copy-back`.
 
+.. note::
+
+    Tables processed during the --prepare step may also be added to the backup even if they are not explicitly listed by the parameter.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst]:
```suggestion
    Tables processed during the --prepare step may also be added to the backup even if they are not explicitly listed by the parameter if they were created after the backup started.
```

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst [hunk 2]
```diff
@@ -74,15 +74,21 @@ this list make sure to specify the ``mysql``, ``sys``, and
...
 :option:`xtrabackup --databases-file` specifies a file that can contain multiple
 databases and tables in the ``databasename[.tablename]`` form, one element name
-per line in the file. Only named databases and tables will be backed up. Names
-are matched exactly, case-sensitive, with no pattern or regular expression
-matching.
+per line in the file. Names are matched exactly, case-sensitive, with no pattern or regular expression matching.
+
+.. note::
+
+    Tables processed during the --prepare step may also be added to the backup even if they are not explicitly listed by the parameter.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst]: Same as above.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst]: Updated.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**